fs/inode/inode.h

```diff
@@ -39,6 +39,8 @@
 #include <nuttx/fs/fs.h>
 #include <nuttx/lib/lib.h>
 
+#include "../fs_heap.h"
```

Comment:
remove ..

Reply:
When referencing inode from a file in a non-fs directory, only the reference path to fs/inode is added. Therefore, if #include "fs_inode.h" is used, the file cannot be searched. This is because the fs directory is not included in the search path. like rpmsgblk_server.c

```
Note: bear 2.4.3 in Ubuntu 20.04 works out of box.
No configuration change.
make: Entering directory '/home/crafcat7/SSD/vela/nuttx.git/nuttx'
CP:  /home/crafcat7/SSD/vela/nuttx.git/nuttx/include/nuttx/config.h
CP:  /home/crafcat7/SSD/vela/nuttx.git/nuttx/include/nuttx/fs/hostfs.h
CC:  libmad/stream.c In file included from misc/rpmsgblk_server.c:34:
/home/crafcat7/SSD/vela/nuttx.git/nuttx/fs/inode/inode.h:42:10: fatal error: fs_heap.h: No such file or directory
   42 | #include <fs_heap.h>
      |          ^~~~~~~~~~~
```